src/agentlab/llm/chat_api.py

```diff
@@ -200,24 +182,41 @@ def make_model(self):
 class ChatModel(ABC):
 
     @abstractmethod
```

Comment:
why is this an abstract method if it's implemented?

Reply:
typing